site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -33,11 +33,12 @@
         $("#queue_filter_" + current_queues[i]).prop("checked", true);
         filter_choices[current_queues[i]] = true;
       }
-      if (filter_choices[current_queues[i]]) {
-          document.getElementById('queue_filter_' + current_queues[i]).parentElement.classList.remove("fully-transparent");
+      var queue_filter = document.getElementById('queue_filter_' + current_queues[i]);
```

Comment:
Try and use [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) and [`const`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) for new code. In this case, since the variable is never re-assigned, we can use `const`.

```suggestion
      const queue_filter = document.getElementById('queue_filter_' + current_queues[i]);
```